### ISSUE TITLE
最新版1.1.1快进会花屏

### ISSUE THREAD

**Issue by zhaochengfeng** [niltsh/niltsh.github.com#3]
非常感谢开发人员。但是我发现一个问题，升级到1.1.1后快进会有短暂的花屏，1.1.0没有问题

**Comment by radellaf** [comment]
I think I'm having the same problem.  In 1.1.1 but not 1.1.0, playback often does not start on an I-frame so you get scrambled video until the next keyframe (can be 3-10 sec).  This sometimes happens right at the beginning of a video.  Also, 60fps playback is still messed up - plays at 30 and has audio sync problems.

**Comment by FakeYin** [comment]
The issue seems to be related to Hardware Acceleration.  
It is solved if I have HA turned off.

**Comment by radellaf** [comment]
Turning of HA works for me as well.  this is the command I used to set that:

**Comment by anzhizou** [comment]
@radellaf Can I turn off HA from MPlayerX GUI?

**Comment by radellaf** [comment]
No, Hardware Acceleration can only be enabled or disabled through the prefs.

http://blog.mplayerx.org/blog/2013/02/01/hidden-settings/

**Comment by anzhizou** [comment]
@radellaf Thanks!

**Comment by radellaf** [comment]
Just noticed this, might help: http://www.macupdate.com/app/mac/50912/mplayerx-settings-gui by Tom Brek

**Comment by torrentprime** [comment]
Stopping hardware acceleration solved the problem for me. Thanks for this!

**Comment by Bluelich** [comment]
How about submit the newest version of MplayerX to Mac App Store?
